app/scripts/filters/util.js

```diff
@@ -348,6 +348,30 @@ angular.module('openshiftConsole')
           .replace(/^./, function(str){ return str.toUpperCase(); });
     };
   })
+  // event messages should simple, but human readable:
+  // Pulled Pod <name>
+  // Unhealthy Pod <name>
+  // Deployment created for DeploymentConfig <name>
+  // etc.
+  // This filter handles the "Reason <optional preposition>"
+  // but not the <name> as we <a> link that in templates.
+  .filter('messageForEvent', function($filter) {
```

Comment:
"Pulled Pod" is just wrong. You're pulling an image.

@serenamarie125 @beanh66 Would you be OK if we moved away from trying to turn the event reasons into sentences? We don't know what all the reasons will be, and I'm worried in many cases the result will be confusing or wrong.